docs/production/deployment.md

```diff
@@ -367,9 +367,20 @@ make the following changes in two configuration files.
    USE_X_FORWARDED_HOST = True
    ```
 
-3. Restart your Zulip server with `/home/zulip/deployments/current/scripts/restart-server`.
+3. Add the following block to `/etc/zulip/zulip.conf`. This will
+   allow Zulip to use the real client IP addresses, instead of
+   the proxy address for all clients. This is important to have
+   accurate addresses on logs, and notification emails such as
+   log-in alerts.
 
-4. Create an Apache2 virtual host configuration file, similar to the
+   ```ini
+   [loadbalancer]
+   ips = 127.0.0.1
+   ```
```

Comment:
We should probably have a sentence spelling out how you determine what IPs to list here.